kymatio/frontend/base_frontend.py

```diff
@@ -0,0 +1,26 @@
+
+class ScatteringBase():
+    def __init__(self):
+        super(ScatteringBase, self).__init__()
+    
+    def build(self):
+        """ Defines elementary routines. 
```

Comment:
For the doc strings in the front ends section do we want to follow the [Numpy](https://numpydoc.readthedocs.io/en/latest/format.html) doc strings convention?

Reply:
We should follow whatever convention is used by the other user-facing functions. I believe it's the same as NumPy, but I'm not 100% certain.